File: 24-experiments/npm-package/experiments.ts
import axios from 'axios';
const API = 'https://api.costkatana.com/api'\;
const KEY = process.env.COST_KATANA_API_KEY;

export async function createExperiment(name: string, variants: any[]) {
  const res = await axios.post(`${API}/experimentation`, {name, variants},
    {headers: {'Authorization': `Bearer ${KEY}`}});
  console.log('✅ Experiment created:', res.data.data.id);
  return res.data.data;
}

File: 24-experiments/python-sdk/experiments.py
import os, requests
API = "https://api.costkatana.com/api"
KEY = os.getenv("COST_KATANA_API_KEY")

def create_experiment(name, variants):
    res = requests.post(f"{API}/experimentation",
        json={"name": name, "variants": variants},
        headers={"Authorization": f"Bearer {KEY}"})
    res.raise_for_status()
    print(f"✅ Experiment created: {res.json()['data']['id']}")
    return res.json()["data"]
